packages/adapter-static/README.md

```diff
@@ -71,6 +71,8 @@ export default {
 
 When operating in SPA mode, you can omit `config.kit.prerender.default` (or set it to `false`, its default value), and only pages that have the [`prerender`](https://kit.svelte.dev/docs/page-options#prerender) option set will be prerendered at build time.
 
+If Node.js throws errors when loading your pages during the build process and this is hard for you to work around, setting `config.kit.prerender.entries` to `[]` will prevent the build process from attempting to load any page or layout components. Do not set `config.kit.prerender.enabled` to `false` because that will also prevent the fallback page from being generated.
```

Comment:
I think we should try to explain _why_ this is happening:

```suggestion
SvelteKit will still crawl your app's entry points looking for prerenderable pages. If `svelte-kit build` fails because of pages that can't be loaded outside the browser, you can set `config.kit.prerender.entries` to `[]` to prevent this from happening. (Setting `config.kit.prerender.enabled` also has this effect, but would prevent the fallback page from being generated.)
```

Reply:
Thanks for the editing! Yes this reads much better.